typescript/enummember.go

```diff
@@ -0,0 +1,30 @@
+package typescript
```

Comment:
On the nitpicky file name topic @vsapronov this should be enum_member at some point